tox.ini

```diff
@@ -37,6 +37,7 @@ commands =
     npm config set spin false
     npm install
     node ./node_modules/gulp/bin/gulp.js build
+    npm run lint
```

Comment:
This works (surprisingly).
That means we could also make an npm script for "build" which just proxies "gulp build" and then shorten line 39 above to just be "npm run build" and shave off a few chars (by adding one level of indirection).